server/sonar-web/src/main/js/apps/organizations/components/OrganizationMembers.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+// @flow
+import React from 'react';
+import PageHeader from './PageHeader';
+import MembersList from './MembersList';
+import UsersSearch from '../../users/components/UsersSearch';
+import ListFooter from '../../../components/controls/ListFooter';
+import type { Organization } from '../../../store/organizations/duck';
+import type { Member } from '../../../store/organizationsMembers/actions';
+
+type Props = {
+  members: Array<Member>,
+  status: { loading?: boolean, total?: number, pageIndex?: number, query?: string },
+  organization: Organization,
+  fetchOrganizationMembers: (organizationKey: string, query?: string) => void,
+  fetchMoreOrganizationMembers: (organizationKey: string, query?: string) => void,
+};
+
+export default class OrganizationMembers extends React.PureComponent {
+  props: Props;
+
+  componentDidMount() {
+    const notLoadedYet = this.props.members.length < 1 || this.props.status.query != null;
```

Comment:
Ok I get it. I was focused on caching the data received from the server and save api calls.